README.rst

```diff
@@ -101,6 +101,20 @@ as long as you see ``Successfully installed`` at the end.
 .. _issue #61:
    https://github.com/IntelPNI/brainiak/issues/61
 
+
+Examples
+========
+
+We include BrainIAK usage examples in the examples directory of the code
```

Comment:
We should point out that this is an example of SRM in BrainIAK. For other components currently we also have example codes available but not in Notebook format.